findfood/settings.py

```diff
@@ -121,6 +122,10 @@
 
 STATIC_URL = '/static/'
 
+STATICFILES_DIRS = (
+    "/Users/Hamim/Desktop/django/findfood/webpage/static_dumps",
```

Comment:
related path should be used here. this will break on anywhere else than your machine